### PR TITLE
Code cleanup and reformatting in System.Collections.Concurrent

### PR DESCRIPTION
**Pull request by jlin177**
Code cleanup and reformatting in System.Collections.Concurrent.

### CODE REVIEW

File: src/System.Collections.Concurrent/src/System/Collections/Concurrent/BlockingCollection.cs [hunk 1]
```diff
@@ -1059,22 +1093,24 @@ private static int TryAddToAnyCore(BlockingCollection<T>[] collections, T item,
         /// <returns>The index which the item has been added, -1 if failed</returns>
         private static int TryAddToAnyFast(BlockingCollection<T>[] collections, T item)
         {
-            for (int i = 0; i < collections.Length; i++)
+            for (int index = 0; index < collections.Length; index++)
```
Comment (stephentoub) [inline src/System.Collections.Concurrent/src/System/Collections/Concurrent/BlockingCollection.cs]: Why did you change this from i to index?  There's nothing in our guidelines against using i... it's very common and reasonable.

File: src/System.Collections.Concurrent/src/System/Collections/Concurrent/BlockingCollection.cs [hunk 2]
```diff
@@ -1090,27 +1126,29 @@ private static List<WaitHandle> GetHandles(BlockingCollection<T>[] collections,
             List<CancellationToken> tokensList = new List<CancellationToken>(collections.Length + 1); // + 1 for the external token
             tokensList.Add(externalCancellationToken);
 
-            //Read the appropriate WaitHandle based on the operation mode.
+            // Read the appropriate WaitHandle based on the operation mode.
             if (isAddOperation)
             {
-                for (int i = 0; i < collections.Length; i++)
+                for (int index = 0; index < collections.Length; index++)
```
Comment (stephentoub) [inline src/System.Collections.Concurrent/src/System/Collections/Concurrent/BlockingCollection.cs]: Ditto.  These changes should be reverted.

File: src/System.Collections.Concurrent/src/System/Collections/Concurrent/BlockingCollection.cs [hunk 3]
```diff
@@ -1090,27 +1126,29 @@ private static List<WaitHandle> GetHandles(BlockingCollection<T>[] collections,
...
-                        tokensList.Add(collections[i]._producersCancellationTokenSource.Token);
+                        handlesList.Add(collections[index]._freeNodes.AvailableWaitHandle);
+                        tokensList.Add(collections[index]._producersCancellationTokenSource.Token);
                     }
                 }
             }
             else
             {
-                for (int i = 0; i < collections.Length; i++)
+                for (int index = 0; index < collections.Length; index++)
```
Comment (stephentoub) [inline src/System.Collections.Concurrent/src/System/Collections/Concurrent/BlockingCollection.cs]: Ditto

File: src/System.Collections.Concurrent/src/System/Collections/Concurrent/BlockingCollection.cs [hunk 4]
```diff
@@ -1714,25 +1778,26 @@ private static void ValidateCollectionsArray(BlockingCollection<T>[] collections
...
-            //The number of WaitHandles must be <= 64 for MTA, and <=63 for STA, and we reserve one for CancellationToken                
             {
-                throw new ArgumentOutOfRangeException(
-nameof(collections), SR.BlockingCollection_ValidateCollectionsArray_LargeSize);
+                // The number of WaitHandles must be <= 64 for MTA, and <=63 for STA, and we reserve one for CancellationToken
+                throw new ArgumentOutOfRangeException(nameof(collections), SR.BlockingCollection_ValidateCollectionsArray_LargeSize);
             }
 
-            for (int i = 0; i < collections.Length; ++i)
+            for (int index = 0; index < collections.Length; ++index)
```
Comment (stephentoub) [inline src/System.Collections.Concurrent/src/System/Collections/Concurrent/BlockingCollection.cs]: Ditto.  I'll stop commenting on these, but please revert other such occurrences.

File: src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentBag.cs [hunk 5]
```diff
@@ -260,16 +264,20 @@ private bool TryTakeOrPeek(out T result, bool take)
                             // Release the lock before stealing
                             if (lockTaken)
                             {
-                                try { }
+                                try
+                                {
+                                }
```
Comment (stephentoub) [inline src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentBag.cs]: This was ok the way it was.

File: src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentBag.cs [hunk 6]
```diff
@@ -685,7 +724,10 @@ public bool IsEmpty
         /// returns false.</value>
         bool ICollection.IsSynchronized
         {
-            get { return false; }
+            get
+            {
+                return false;
+            }
```
Comment (stephentoub) [inline src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentBag.cs]: This was ok the way it was.  Please revert.

File: src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentBag.cs [hunk 7]
```diff
@@ -810,16 +856,19 @@ private void WaitAllOperations()
             Debug.Assert(Monitor.IsEntered(GlobalListsLock));
 
             ThreadLocalList currentList = _headList;
+
             while (currentList != null)
             {
                 if (currentList._currentOp != (int)ListOperation.None)
                 {
                     SpinWait spinner = new SpinWait();
+
```
Comment (stephentoub) [inline src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentBag.cs]: Why are you adding all of these blank lines in places?  Please revert.

File: src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentDictionary.cs [hunk 8]
```diff
@@ -1230,7 +1338,10 @@ public bool IsEmpty
         /// <see cref="T:System.Collections.Generic.Dictionary{TKey,TValue}"/>.</value>
         public ICollection<TKey> Keys
         {
-            get { return GetKeys(); }
+            get
+            {
+                return GetKeys();
+            }
```
Comment (stephentoub) [inline src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentDictionary.cs]: This was fine the way it was. Please revert.

File: src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentDictionary.cs [hunk 9]
```diff
@@ -1241,7 +1352,10 @@ public ICollection<TKey> Keys
         /// the <see cref="T:System.Collections.Generic.IReadOnlyDictionary{TKey,TValue}"/>.</value>
         IEnumerable<TKey> IReadOnlyDictionary<TKey, TValue>.Keys
         {
-            get { return GetKeys(); }
+            get
+            {
+                return GetKeys();
+            }
```
Comment (stephentoub) [inline src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentDictionary.cs]: Ditto on all of these get accessors.

File: src/System.Collections.Concurrent/src/System/Collections/Concurrent/PartitionerStatic.cs [hunk 10]
```diff
@@ -188,46 +192,74 @@ public static OrderablePartitioner<TSource> Create<TSource>(IEnumerable<TSource>
...
+            return Create(CreateRanges(fromInclusive, toExclusive, rangeSize), EnumerablePartitionerOptions.NoBuffering); // chunk one range at a time
         }
 
         // Private method to parcel out range tuples.
         private static IEnumerable<Tuple<long, long>> CreateRanges(long fromInclusive, long toExclusive, long rangeSize)
         {
             // Enumerate all of the ranges
-            long from, to;
+            long from;
+            long to;
```
Comment (stephentoub) [inline src/System.Collections.Concurrent/src/System/Collections/Concurrent/PartitionerStatic.cs]: Please revert.

File: src/System.Collections.Concurrent/src/System/Collections/Concurrent/PartitionerStatic.cs [hunk 11]
```diff
@@ -188,46 +192,74 @@ public static OrderablePartitioner<TSource> Create<TSource>(IEnumerable<TSource>
...
             for (long i = fromInclusive; (i < toExclusive) && !shouldQuit; i += rangeSize)
             {
                 from = i;
-                try { checked { to = i + rangeSize; } }
+                try
+                {
+                    checked
+                    {
+                        to = i + rangeSize;
+                    }
```
Comment (stephentoub) [inline src/System.Collections.Concurrent/src/System/Collections/Concurrent/PartitionerStatic.cs]: It's fine to have this on one line; please revert.

File: src/System.Collections.Concurrent/src/System/Collections/Concurrent/PartitionerStatic.cs [hunk 12]
```diff
@@ -278,39 +326,51 @@ public static OrderablePartitioner<TSource> Create<TSource>(IEnumerable<TSource>
             for (int i = fromInclusive; (i < toExclusive) && !shouldQuit; i += rangeSize)
             {
                 from = i;
-                try { checked { to = i + rangeSize; } }
+                try
+                {
+                    checked
+                    {
+                        to = i + rangeSize;
+                    }
```
Comment (stephentoub) [inline src/System.Collections.Concurrent/src/System/Collections/Concurrent/PartitionerStatic.cs]: Ditto

**Comment by jlin177** [comment]
Thanks for your comments! @stephentoub 

The purpose of these changes is to make code more readable. Having line breaks between compound statements increases readability. I'll revert these changes if it's a problem.

Thanks!

**Comment by stephentoub** [comment]
> The purpose of these changes is to make code more readable. Having line breaks between compound statements increases readability.

For me personally, many of the blank lines you added actually decreases the readability.  There may be a few that help, but overall I find it to be a step backwards.  Please revert them.  Thanks.

**Comment by jlin177** [comment]
@stephentoub 
Sure. Will revert these changes. Thanks!
